lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSlide.js

```diff
@@ -35,20 +41,22 @@ class CampaignSlide extends React.Component {
           <div className="wrapper">
             {this.renderHeader()}
 
-            {this.campaigns.map(campaign => {
-              // Fix for local files / data missing by preventing null error
-              const image_uri = campaign.cover_image.default != null ? campaign.cover_image.default.sizes.landscape.uri : '';
+            <div className="gallery -triad">
+              {this.state.campaigns.map(campaign => {
+                // Fix for local files / data missing by preventing null error
+                const image_uri = campaign.cover_image.default != null ? campaign.cover_image.default.sizes.landscape.uri : '';
 
-              const props = {
-                id: campaign.id,
-                image_uri: image_uri,
-                tagline: campaign.tagline,
-                title: campaign.title,
-                signup: this.controller.signup.bind(this.controller)
-              }
+                const props = {
+                  id: campaign.id,
+                  image_uri: image_uri,
+                  tagline: campaign.tagline,
+                  title: campaign.title,
+                  signup: this.controller.signup.bind(this.controller)
+                }
 
-              return <CampaignSignupCard {...props} key={campaign.id} />
-            })}
+                return <CampaignSignupCard {...props} key={campaign.id} />
```

Comment:
I think it reads a little better to just specify the props here, rather than forming a temporary object above and applying it with a spread. It's okay if the element wraps to multiple lines, for example:

```jsx
return (
  <CampaignSignupCard id={campaign.id} image_uri={image_uri} tagline={campaign.tagline}
    key={campaign.id} title={campaign.title} signup={this.controller.signup} />
);
```